Lib/test/_test_embed_structseq.py

```diff
@@ -1,29 +1,28 @@
 import sys
 import types
-import unittest
 
 
 # bpo-46417: Test that structseq types used by the sys module are still
 # valid when Py_Finalize()/Py_Initialize() are called multiple times.
```

Comment:
Please also add a comment explaining why this can't use unittest.

Reply:
Done!